conf.py

```diff
@@ -2,8 +2,11 @@
 
 # -- Project information -----------------------------------------------------
 
+from datetime import datetime
+
+current_year = datetime.today().year
 project = 'Omnivector Slurm Distribution'
-copyright = '2021, Omnivector Solutions'
+copyright = f'2021-{current_year}, Omnivector Solutions'
```

Comment:
is it common to write the year this way? Can you give external examples?

Reply:
Example: go to https://www.python.org/ and scroll all to the bottom of the page.